### PR TITLE
Homework 11

### PR DESCRIPTION
**Pull request by DimitryusX**

### CODE REVIEW

File: Homework11/Homework11.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        {6, 7, 8, 9, 10},
+        {11, 12, 13, 14, 15},
+        {16, 17, 18, 19, 20}
+    };
+
+    const size_t size = sizeof(arrayOfNumbers) / sizeof(int);
+
+    int numberToFind = 10;
+    std::cout << "Let's find number " << numberToFind << std::endl;
+    bool result = find(&arrayOfNumbers[0][0], size, numberToFind);
```
Comment (helen-black) [inline Homework11/Homework11.cpp:24]: find(arrayOfNumbers, size, numberToFind);

File: Homework11/Homework11.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    std::cout << "Let's find number " << numberToFind << std::endl;
+    result = find(arrayOfNumbers, numberToFind);
+    std::cout << "Search result: " << std::boolalpha << result << std::endl;
+
+    std::cout << std::endl << "Task #2" << std::endl << std::endl;
+
+    int ascendingArray[COLS]{ 1, 2, 3, 4, 5 };
+    int descentingArray[COLS]{ 5, 4, 3, 2, 1 };
+
+    bool isSorderdResult = isSorted(&ascendingArray[0], COLS, SortingDirection::ascending);
```
Comment (helen-black) [inline Homework11/Homework11.cpp:42]: the same, ascendingArray

File: Homework11/Utils.cpp [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include "Utils.h"
+
+bool find(const int arr_2d[4][5], const int toFind) {
```
Comment (helen-black) [inline Homework11/Utils.cpp:4]: use constants here

File: Homework11/Utils.cpp [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    for (int i = 0; i < 4; i++) {
+        for (int j = 0; j < 5; j++) {
```
Comment (helen-black) [inline Homework11/Utils.cpp:6]: the same

**Review by helen-black — CHANGES_REQUESTED** [review]

File: Homework11/Homework11.cpp [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+#include "utils.h"
+
+int main()
+{
+    std::cout << "Welcome to homework 11!" << std::endl;
+
+    std::cout << std::endl << "Task #1" << std::endl << std::endl;
+
+    const size_t ROWS = 4;
+    const size_t COLS = 5;
```
[the remark this answers is not in the document]
Comment (DimitryusX) [reply Homework11/Homework11.cpp]: Fixed.

File: Homework11/Homework11.cpp (hunk 2 above)
Comment (DimitryusX) [reply Homework11/Homework11.cpp]: Fixed.

File: Homework11/Utils.cpp (hunk 3 above)
Comment (DimitryusX) [reply Homework11/Utils.cpp]: Fixed.

File: Homework11/Utils.cpp (hunk 4 above)
Comment (DimitryusX) [reply Homework11/Utils.cpp]: Fixed.

File: Homework11/Homework11.cpp (hunk 1 above)
Comment (DimitryusX) [reply Homework11/Homework11.cpp:21]: @helen-black 
I don't know how I can pass a two-level array to function without a pointer or const value for rows and cols (my main idea was passing an array with minimum parameters).

**Pull request merged**